### DISCUSSION TITLE
Nexstim TMS Simulation - Coordinates

### DISCUSSION THREAD

**Discussion by prem-ganesh23** [simnibs/simnibs#411] [8. Simulation workflows] [answered]
Hello,
I'm trying to model TMS simulations using coordinates exported from a Nexstim TMS system. Nexstim outputs three coordinates for coil location, direction, and normal in mm. However, I've noticed that these coordinates are wildly different from those that SimNIBS gives me when I manually place a point on the GUI as close to the center of the coil. Both are in mm. I have the below information from Nexstim about their coordinate system:

MRI coordinates are defined from the subject's MR images. The origin of the MRI coordinate space is the bottom corner of the MR image set behind the right ear of the subject:

· the x-axis points from the origin to the left ear

· the y-axis points from the origin up to the top of the head

· the z-axis points from the origin to the front of the head.

The stimulation is done based on a DICOM file, which was converted to NIFTI for meshing using charm. However, when I replicate the same points on the Nexstim using the same nifti file as meshing the coordinates still do not match up.

**Comment by diedrim** [comment, marked as answer]
SimNIBS uses the q-form of the NiFTI file to define the subject coordinate system. This seems to be a different coordinate space than the one defined by Nexstim. You need to convert your positions accordingly.
